File: Frontend/Logic/B20/Architecture/Events/Context/Impl.cs
using B20.Architecture.Contexts.Api;
using B20.Events.Api;
using B20.Events.Impl;

namespace B20.Architecture.Events.Context
{
    public class EventsImpl: ContextModule
    {
        public void Apply(ContextBuilder builder)
        {
            builder.SetImpl<EventPublisher, EventPublisherLogic>();
        }
    }
}

File: Backend/lib/src/main/kotlin/com/github/bratek20/ostium/singlegame/context/Impl.kt
package com.github.bratek20.ostium.singlegame.context

import com.github.bratek20.architecture.context.api.ContextBuilder
import com.github.bratek20.architecture.context.api.ContextModule

import com.github.bratek20.ostium.singlegame.api.*
import com.github.bratek20.ostium.singlegame.impl.*

class GameModuleImpl: ContextModule {
    override fun apply(builder: ContextBuilder) {
        builder
            .setImpl(SingleGameApi::class.java, SingleGameApiLogic::class.java)
    }
}